tests/gold_tests/pluginTest/combo_handler/combo_handler.test.py

```diff
@@ -104,12 +106,12 @@ def add_server_obj(content_type, path):
     'map http://localhost/sub/ http://127.0.0.1:{}/sub/'.format(server.Variables.Port)
 )
 
-ts.Disk.File(ts.Variables.CONFIGDIR + "/ctwl.txt", id="ctwl_cfg", typename="ats:config")
-ts.Disk.ctwl_cfg.AddLine("# test  ")
-ts.Disk.ctwl_cfg.AddLine("")
-ts.Disk.ctwl_cfg.AddLine("		text/javascript  # test side comment")
-ts.Disk.ctwl_cfg.AddLine("  application/javascript")
-ts.Disk.ctwl_cfg.AddLine("text/css")
+# Configure the combo_handler's configuration file.
+ts.Setup.Copy("ctwl.txt", ts.Variables.CONFIGDIR)
+ctwl_cfg_path = os.path.join(ts.Variables.CONFIGDIR, "ctwl.txt")
+ts.Setup.Chown(ctwl_cfg_path, "nobody", "nobody", ignore=True)
+# covers ubuntu's unprivileged group
+ts.Setup.Chown(ctwl_cfg_path, "nobody", "nogroup", ignore=True)
```

Comment:
I am a little worried about this as we found issues where nobody or nogroup causes errors.

Reply:
This was needed because the esi plugin used default mode of read and write. Fixes the plugin obviates the need for these chowns.